src/runtime-rs/crates/shimv2/src/bin/shimv2.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
```

Comment:
not super familiar with rust/crate conventions, is having a source file in `bin/` common?

Reply:
Currently, add to all `.rs` files